packages/framework/src/Model/ImageSitemap/ImageSitemapItemImage.php

```diff
@@ -12,7 +12,7 @@ class ImageSitemapItemImage
 
     public ?string $geoLocation = null;
 
-    public string $title;
+    public ?string $title = null;
```

Comment:
Does it make sense to keep and populate the `title` field if it's not used? 

According to the [documentation](https://developers.google.com/search/docs/crawling-indexing/sitemaps/image-sitemaps), the `caption`, `geo_location`, `title`, and `license` are deprecated. Wouldn't make sense to remove them all completely?

Reply:
You're right, fields caption, geo_location, license are deprecated too, so i removed them all completely.